specifyweb/frontend/js_src/lib/components/Forms/BaseResourceView.tsx

```diff
@@ -57,7 +57,7 @@ export function useResourceView<SCHEMA extends AnySchema>({
             if (resource === undefined) return undefined;
             format(resource)
               .then((title) => {
-                setFormatted(title ?? '');
+                setFormatted(title ?? ('' as LocalizedString));
```

Comment:
This change is not necessary.
The error here is created by your ide being misconfigured. It's not an actual typescript error